canopen/objectdictionary/__init__.py

```diff
@@ -407,6 +415,8 @@ def encode_raw(self, value: Union[int, float, str, bytes, bytearray]) -> bytes:
         elif self.data_type == UNICODE_STRING:
             # Is this correct?
```

Comment:
I'd assume this *IS* correct.  UNSIGNED16 is defined as little-endian and an array of that equals UNICODE_STRING.  Could remove the question.

Reply:
Canopen does not specify _which_ 16-bit encoding to use, only that the glyphs are 16-bit little-endian symbols, so there is design choice taken here.

Reply:
Right, it doesn't mention UTF-16 explicitly.  But it's definitely the most common choice for 16-bit unicode.  And the little-endian part is clear from the fact that it's UNSIGNED16 in CANopen.

So I think the comment can go, let's wait for the first user to hit any other encoding before we start thinking about a possible alternative.